client/gutenberg/extensions/shared/render-material-icon.jsx

```diff
@@ -0,0 +1,15 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+const renderMaterialIcon = content => (
```

Comment:
I agree but wanted to keep this PR simple; I almost didn't move the file because of it. :-)

Happy to rename unless @enejb has something to add on why it should be called "material" and kept at "jetpack preset" -utils.

Relevant: https://github.com/Automattic/wp-calypso/issues/28580 _"Import material icons from library"_

Reply:
Sorry, that wasn't a critique of changes in this PR or a request for changes. Excuse the existential questions 😅

Reply:
> But why make this Material specific? 

Naming things is hard.

Reply:
💯 it is!